### PR TITLE
Credorax: Add Gateway

### PR DESCRIPTION
**Pull request by aellispierce**

### CODE REVIEW

**Comment by anellis** [comment]
@rwdaigle or @duff Would you mind taking a look here? 

We need to call them and have them approve the integration before getting the production URL and, while they have support for store, it's not active on our test account yet so I skipped it. We can certainly add it later, but since my days are numbered here I wanted to go ahead and get the basic integration out of the way. Didn't want to be waiting on their customer support to give us the URL or turn store on for our account.

File: lib/active_merchant/billing/gateways/credorax.rb [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+      def commit(action, params, reference_action = nil)
+        raw_response = ssl_post(url, post_data(action, params, reference_action))
+        response = parse(raw_response)
+
+        Response.new(
+          success_from(response),
+          message_from(response),
+          response,
+          authorization: "#{response["Z1"]};#{response["Z4"]};#{response["A1"]};#{action}",
```
Comment (rwdaigle) [inline lib/active_merchant/billing/gateways/credorax.rb]: ugh - all of these are required to properly correlate future reference txs? :crying_cat_face:

**Comment by rwdaigle** [comment]
@anellis this looks great, thanks! We'll take it from here.

**Comment by davidsantoso** [comment]
@aellispierce thanks for the starting point! Added in https://github.com/activemerchant/active_merchant/commit/b2f62a950faab76d1e0417fd090602ae5ac3363b.

**Comment by rwdaigle** [comment]
@aellispierce 👋 😝

**Comment by aellispierce** [comment]
All my hard work, stolen right out from under me!

**Comment by davidsantoso** [comment]
@aellispierce I'd feel bad, except I had to do/go through their certification process so I feel less sorry about stealing your commit 😸

**Comment by duff** [comment]
This is like a slack channel! 👋 @aellispierce

**Comment by aellispierce** [comment]
👋 @duff! @davidsantoso This will not be forgotten. 👿

**Comment by duff** [comment]
Hoping @davidsantoso doesn't start seeing "issues" with his Github account.

**Comment by davidsantoso** [comment]
@aellispierce please! Spare my account! 🙏

**Pull request closed without merging**